RELEASING.md

```diff
@@ -188,6 +188,28 @@ Central (the staging repository will be destroyed in the process). You can see
 the complete process for releasing to Maven Central on the [OSSRH
 site](http://central.sonatype.org/pages/releasing-the-deployment.html).
 
+### Update JavaDoc
+
+```bash
+$ git checkout -b update-javadoc gh-pages
```

Comment:
"upstream/gh-pages" ?

Reply:
Agree. "gh-pages" is not recognized and need to explicitly specify remote.